src/webviews/components/issue/JiraIssuePage.tsx

```diff
@@ -910,36 +912,43 @@ export default class JiraIssuePage extends AbstractIssueEditorPage<Emit, Accept,
 
         return (
             <Page>
-                <WidthDetector>
-                    {(width?: number) => {
-                        if (width && width < 800) {
-                            return (
-                                <div style={{ margin: '20px 16px 0px 16px' }}>
-                                    {this.getMainPanelHeaderMarkup()}
-                                    {this.commonSidebar()}
-                                    {this.getMainPanelBodyMarkup()}
-                                    {this.advancedSidebar()}
-                                    {this.createdUpdatedDates()}
-                                </div>
-                            );
-                        }
-                        return (
-                            <div style={{ maxWidth: '1200px', margin: '20px auto 0 auto' }}>
-                                <Grid layout="fluid">
-                                    <GridColumn medium={8}>
+                <AtlascodeErrorBoundary
+                    context={{ view: AnalyticsView.JiraIssuePage }}
+                    postMessageFunc={(e) => {
+                        this.postMessage(e); /* just {this.postMessage} doesn't work */
```

Comment:
Apparently just putting `this.postMessage` here doesn't work. I'm guessing it's because postMessage is not a callback, and not memo-ized properly? I'd really rather not mess with these pages more than absolutely necessary :D